server/controllers/customer.js

```diff
@@ -3,8 +3,10 @@ import {extend, intersection} from 'lodash'
 import {ForbiddenError, NotFoundError} from '../lib/errors'
 import {ADMIN_ROLE, clientRoles, volunteerRoles} from '../../common/constants'
 import Customer from '../models/customer'
+import Volunteer from '../models/volunteer'
```

Comment:
This isn't used, delete it. 

Do the tests not run when you `git commit`? It should be a lint error and prevent committing. Can you run `npm run lint`?